NDTensors/src/blocksparse/linearalgebra.jl

```diff
@@ -184,9 +184,12 @@ function LinearAlgebra.svd(T::BlockSparseMatrix{ElT}; kwargs...) where {ElT}
     # This sign (sVP) accounts for the fact that
     # V is transposed, i.e. the index connecting to S
     # is the second index:
-    sVP = using_auto_fermion() ? -block_parity(vind, blockV[2]) : 1
+    sVP = 1
+    if using_auto_fermion()
+      sVP = block_parity(vind, blockV[2]) == 1 ? -1 : +1
```

Comment:
Maybe could be `sVP = -block_parity(vind, blockV[2])`.

Reply:
The issue with the previous code was that `block_parity` outputs 0 or 1, not +1 or -1. We could define a new function maybe called `block_sign` though.

Reply:
`block_sign` sounds good.